vars/infra.groovy

```diff
@@ -141,8 +141,14 @@ Object withArtifactCachingProxy(boolean useArtifactCachingProxy = true, Closure
   if (useArtifactCachingProxy) {
     echo "INFO: using artifact caching proxy from '${requestedProxyProvider}' provider."
     configFileProvider(
-        [configFile(fileId: "artifact-caching-proxy-${requestedProxyProvider}", variable: 'MAVEN_SETTINGS')]) {
-          body()
+        [configFile(fileId: "artifact-caching-proxy-${requestedProxyProvider}", targetLocation: '/tmp/settings.xml')]) {
```

Comment:
Why the switch to a hard-coded location? Fine for a container agent but could break concurrent activity on some agents.

Reply:
I'm not really happy with that too but I had issues with the variable, I didn't managed to retrieve its value while running the unit tests.
Will try again.

Reply:
> could break concurrent activity on some agents.

Maybe adding a random value and/or an agent identifier would avoid breaking concurrent activity or is it simply a bad idea?
I need to check that more thoroughly tomorrow.